Libraries/LibJS/Parser.cpp

```diff
@@ -2073,7 +2079,7 @@ NonnullRefPtr<ObjectExpression const> Parser::parse_object_expression()
                 invalid_object_literal_property_range = expression->source_range();
         } else if (match(TokenType::ParenOpen)) {
             VERIFY(property_key);
-            u8 parse_options = FunctionNodeParseOptions::AllowSuperPropertyLookup;
+            u16 parse_options = FunctionNodeParseOptions::AllowSuperPropertyLookup;
```

Comment:
Why is this not using our the enum directly
(Granted marking this enum as having bitwise ops, which we have a nice helper for)

Reply:
That turns into a big messy change. I'll let it be for now :)